2015-09-08 17:45:02:376 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:45:11:521 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:45:11:644 | Loginator.App | Error | [OnStartup] An unhandled dispatcher exception occurred. | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2015-09-08 17:45:58:417 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:46:35:487 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:46:41:584 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:46:44:513 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:46:48:880 | Loginator.App | Error | [OnStartup] An unhandled dispatcher exception occurred. | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2015-09-08 17:49:50:295 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:49:56:082 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:49:56:198 | Loginator.App | Error | [OnStartup] An unhandled dispatcher exception occurred. | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2015-09-08 17:49:57:369 | Loginator.App | Fatal | [OnStartup] An unhandled exception occurred and the application is terminating | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at System.Windows.Application.Run()
   at Loginator.App.Main() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\obj\Debug\App.g.cs:line 0
2015-09-08 17:50:18:195 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:50:25:747 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:50:29:557 | Loginator.App | Error | [OnStartup] An unhandled dispatcher exception occurred. | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2015-09-08 17:51:15:888 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:51:18:405 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:51:20:395 | Loginator.App | Error | [OnStartup] An unhandled dispatcher exception occurred. | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2015-09-08 17:51:34:225 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:51:36:406 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:51:36:526 | Loginator.App | Error | [OnStartup] An unhandled dispatcher exception occurred. | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2015-09-08 17:51:40:734 | Loginator.App | Info | [OnStartup] Application successfully started | 
2015-09-08 17:51:40:833 | Loginator.App | Error | [OnStartup] An unhandled dispatcher exception occurred. | System.Windows.Markup.XamlParseException: 'The invocation of the constructor on type 'Loginator.MainWindow' that matches the specified binding constraints threw an exception.' Line number '4' and line position '9'. ---> System.TypeInitializationException: The type initializer for 'LogApplication.ViewModels.LoginatorViewModel' threw an exception. ---> Common.Exceptions.LoginatorException: Port 7071 is already in use
   at Backend.Receiver.Initialize(Configuration configuration) in c:\Users\Daniel\Documents\GitHub\Loginator\Backend\Receiver.cs:line 58
   at LogApplication.ViewModels.LoginatorViewModel..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 103
   at LogApplication.ViewModels.LoginatorViewModel..cctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\ViewModels\LoginatorViewModel.cs:line 24
   --- End of inner exception stack trace ---
   at LogApplication.ViewModels.LoginatorViewModel.get_Instance()
   at Loginator.MainWindow..ctor() in c:\Users\Daniel\Documents\GitHub\Loginator\Loginator\MainWindow.xaml.cs:line 34
   --- End of inner exception stack trace ---
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
   at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
   at System.Windows.Application.DoStartup()
   at System.Windows.Application.<_ctor>b__0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
2015-09-08 17:52:45:692 | Loginator.App | Info | [OnStartup] Application successfully started | 
